pyspectrafuse/common/sdrf_utils.py

```
@@ -25,53 +26,55 @@ def get_sdrf_file_path(folder: str) -> str:
-            ValueError: If instrument field cannot be parsed
+    def get_metadata_dict_from_sdrf(sdrf_folder: str, fill_unknown=True):
         """
-        # print("reading sdrf file: {}".format(sdrf_folder))
-        sdrf_df = pd.read_csv(sdrf_folder, sep='\t')
-        sdrf_feature_df = pd.DataFrame()
-        try:
-            sdrf_feature_df = sdrf_df.loc[:, ['comment[data file]', 'Characteristics[organism]', 'comment[instrument]']]
-        except KeyError as e:
-            missing_cols = set(['comment[data file]', 'Characteristics[organism]', 'comment[instrument]']) - set(sdrf_df.columns)
-            raise KeyError(f'{sdrf_folder} file has format error. Missing columns: {missing_cols}. Available columns: {list(sdrf_df.columns)}') from e
+        从SDRF DataFrame中构建样本信息字典：{数据文件名: [物种, 仪器名称]}
```

Ruff (0.15.6)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)